Updated london event info (#2964)
nf-core/website: sites/main-site/src/content/events/2025/hackathon-march-2025/university-college-london.mdx

endDate: '2025-03-26'
endTime: '17:00+01:00'
locations:
    - name: University College London (East Campus, Stratford)
      address: |
          Community Classroom, Marshgate Building, 7 Sidings St, London E20 2AE, UK
      links:
          - https://www.ucl.ac.uk

>       geoCoordinates: [51.52188666324138, -0.13513104474065718]

Reviewer: ```suggestion
      geoCoordinates: [51.537380655501195, -0.011572112473847292]
```

Attendees from outside University College London **can** attend.

> Main contact:
> Chris Wyatt ([ucbtcdr@ucl.ac.uk](mailto:ucbtcdr@ucl.ac.uk)),
> Fernando Duarte ([ucbtfrd@ucl.ac.uk](mailto:ucbtfrd@ucl.ac.uk))

Reviewer: ```suggestion
Main contacts:

- Chris Wyatt ([ucbtcdr@ucl.ac.uk](mailto:ucbtcdr@ucl.ac.uk)),
- Fernando Duarte ([ucbtfrd@ucl.ac.uk](mailto:ucbtfrd@ucl.ac.uk))
```

## Site Specific guidelines

We are hosting a site at the new UCL EAST campus in Stratford.

It has a capacity of 22 people. We hope to have lunch sponsored on one of the days, plus refreshments.

Registration is required, and will open on the January 31st.

Welcome to all!
(with limits on numbers of people, first come, first served policy).

More information to follow soon. Hope you can make it!
It is a great place to meet with fellow London (and further) based Nextflow enthuiasts

## Guidelines

Please familiarize yourself with the [hackathon guidelines](https://nf-co.re/events/2025/hackathon-march-2025#in-person-local-sites)

## Alternative overflow location (TBC)

> In case we get a huge response, we have an alternative additional option at the Bloomsbury campus (TBD).
> 
> **24th:** 188 Tottenham Ct Rd (Room 4). Capacity = 40
> 
> **25th:** LG04 Otto Koenigsberger & LG02 Patrick Abercrombie, Central House. Capacity = 20
> 
> **26th:** 302 & 402 Tutorial Room, Bentham House. Capacity = 15

Reviewer: ```suggestion
In case we get a huge response, we have an alternative additional options at the Bloomsbury campus (TBC):

- **24th:** 188 Tottenham Ct Rd (Room 4). Capacity = 40
- **25th:** LG04 Otto Koenigsberger & LG02 Patrick Abercrombie, Central House. Capacity = 20
- **26th:** 302 & 402 Tutorial Room, Bentham House. Capacity = 15
```
